README.md

```diff
@@ -55,7 +55,7 @@ The Budget Tracker app is a mobile web application where you can manage your bud
 
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your branch name more descriptive because the branch name you have is not descriptive enough 😨 You could have used feature/finalize-budget-tracker which is more descriptive than style 👍